### DISCUSSION TITLE
Can Ensemble handle receiving sharing intent?

### DISCUSSION THREAD

**Discussion by JasPanesar** [EnsembleUI/ensemble#157] [Q&A] [answered]
Hi, hopefully a simple Q:

I'd like to be able to share in a link, video, photo, PDF, etc into an Ensemble app.

Is there ready support for this in Ensemble? 

f not, is there a way that his could be reasonably achieved with a combination of the current Ensemble functionality plus additional custom code? (per platform, I'm assuming).  

https://pub.dev/packages/receive_sharing_intent

**Comment by kmahmood74** [comment, marked as answer]
@JasPanesar this is now available.

**Reply by JasPanesar** [reply]
Fantastic! This is a game changer for regular ingress and egress.

**Reply by kmahmood74** [reply]
to try it - download the Ensemble Go (iOS) or Ensemble Preview (android). Then select Kitchen Sink -> Actions & Events -> Share

To see the code - https://studio.ensembleui.com/app/e24402cb-75e2-404c-866c-29e6c3dd7992/screen/Dnv8CceAHCHlEpS61DEE

**Reply by kmahmood74** [reply]
go to our website - https://ensembleui.com it has QR code / links to download the Ensemble preview apps
